Documentation/getting-started.md

```diff
@@ -20,18 +20,22 @@ Getting started
 git clone https://github.com/stratisproject/StratisBitcoinFullNode.git  
 ```
 
-3. Checkout the develop branch (or master without latest changes)
+3. Checkout the master branch
 ```
 cd StratisBitcoinFullNode
-git checkout develop
+git checkout master
```

Comment:
if they just cloned the repo, checking out the master branch seems redundant, although it won't hurt for them to do so.  In the existing instructions it made sense to have a step to check out the develop branch since it wasn't the default branch for the repo.